config/config.go

```diff
@@ -135,6 +135,7 @@ func init() {
 	cfg.SetDefault("analyzer.topology.k8s.config_file", "/etc/skydive/kubeconfig")
 	cfg.SetDefault("analyzer.topology.ovn.address", "unix:///var/run/openvswitch/ovnnb_db.sock")
 	cfg.SetDefault("analyzer.topology.istio.config_file", "/etc/skydive/kubeconfig")
+	cfg.SetDefault("analyzer.startup.capture_type", "")
```

Comment:
You can simply remove the line

Reply:
Removed in 1222b9696ed92681b5a29536e4029cd42bd96694 .